### PR TITLE
Added auth key insertion to ovirt module

### PR DESCRIPTION
**Pull request by joshainglis**
Added functionality to ovirt module to allow insertion of authorised key on vm instantiation.
Minor formatting changes where code has been touched.

### CODE REVIEW

File: cloud/misc/ovirt.py [hunk 1]
```diff
@@ -151,6 +151,18 @@
     required: false
     aliases: []
     choices: ['present', 'absent', 'shutdown', 'started', 'restarted']
+  authorised_key_file:
+    description:
```
Comment (bcoca) [inline cloud/misc/ovirt.py:155]: needs version_added (2.0)

File: cloud/misc/ovirt.py [hunk 2]
```diff
@@ -151,6 +151,18 @@
...
+     - absolute path to the public key to be inserted into authorised keys on instantiation
+    default: null
+    required: false
+    aliases: []
```
Comment (bcoca) [inline cloud/misc/ovirt.py:159]: no need for empty aliases

File: cloud/misc/ovirt.py [hunk 3]
```diff
@@ -151,6 +151,18 @@
     required: false
     aliases: []
     choices: ['present', 'absent', 'shutdown', 'started', 'restarted']
+  authorised_key_file:
```
Comment (bcoca) [inline cloud/misc/ovirt.py:154]: authorized?

**Comment by joshainglis** [comment]
Closing this as it is subsumed by #589

**Pull request closed without merging**